src/config/dark-sites.config

```diff
@@ -328,6 +328,7 @@ hoppscotch.io
 hostedtalk.net
 hotstar.com
 htmlpasta.com
+hulu.com
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/109141822-13528500-7756-11eb-848d-d1fa37c132ba.png)
Doesn't seem that dark to me.

Reply:
If you have a hulu account and log in the entire site is dark
<img width="1429" alt="image" src="https://user-images.githubusercontent.com/73370925/109157219-bd72e280-773f-11eb-8af7-00009d5b0019.png">